docs/docsite/rst/user_guide/playbooks_reuse.rst

```diff
@@ -24,23 +24,23 @@ Ansible has two modes of operation for reusable content: dynamic and static.
 
 In Ansible 2.0, the concept of *dynamic* includes was introduced. Due to some limitations with making all includes dynamic in this way, the ability to force includes to be *static* was introduced in Ansible 2.1. Because the *include* task became overloaded to encompass both static and dynamic syntaxes, and because the default behavior of an include could change based on other options set on the Task, Ansible 2.4 introduces the concept of ``include`` vs. ``import``.
 
-If you use any ``import*`` Task (``import_playbook``, ``import_tasks``, etc.), it will be *static*.
 If you use any ``include*`` Task (``include_tasks``, ``include_role``, etc.), it will be *dynamic*.
+If you use any ``import*`` Task (``import_playbook``, ``import_tasks``, etc.), it will be *static*.
```

Comment:
we should distinguish `include_*` and `import_*` from `include:` since this latter one does both ... sometimes .. depending on different options and factors in unclear ways ... which is why we created the include/import_ options.

Reply:
> we should distinguish `include_*` and `import_*` from `include:` since this latter one does both ... sometimes .. depending on different options and factors in unclear ways ... which is why we created the include/import_ options.

@bcoca - I've re-submitted this PR here: https://github.com/ansible/ansible/pull/58998 , I was editing the wrong branch and didnt realise before submitting this PR.  Sorry!  To avoid splitting the conversation I'll reply to you on that PR, apologies for the confusion :)